pilot/pkg/trustbundle/trustbundle_test.go

```diff
@@ -36,9 +43,56 @@ var (
 	rootCACert         string = readCertFromFile(path.Join(env.IstioSrc, "samples/certs", "root-cert.pem"))
 	nonCaCert          string = readCertFromFile(path.Join(env.IstioSrc, "samples/certs", "workload-bar-cert.pem"))
 	intermediateCACert string = readCertFromFile(path.Join(env.IstioSrc, "samples/certs", "ca-cert.pem"))
+
+	validSpiffeX509Bundle = `
```

Comment:
please comment how this was created and how to recreate it if needed

Reply:
Unsure how this is generated. Since these changes are relying on spiffe package functionality to retrieve certBundles, I am thinking I will just add a comment indicating this variable was borrowed from spiffe_test.go?